rllib/evaluation/worker_set.py

```diff
@@ -682,7 +677,7 @@ def valid_module(class_path):
             log_level=config["log_level"],
             callbacks=config["callbacks"],
             input_creator=input_creator,
-            off_policy_estimation_methods=off_policy_estimation_methods,
+            off_policy_estimation_methods=config["off_policy_estimation_methods"],
```

Comment:
makes sense to me -- should be able to use OPE even if the input is a sampler (for debug purposes, or if working with a separate eval ds)

Reply:
Yup, only thing that probably doesn't work currently is mixed input e.g. ("sampler" + json)